packages/server/package.json

```diff
@@ -40,7 +40,7 @@
         "compile": "tsc -b tsconfig.json",
         "acceptancetest": "ts-mocha ./tests/acceptance.spec.ts",
         "start": "node dist/index.js",
-        "test": "nyc ts-mocha --recursive ./tests/*.spec.ts"
+        "test": "nyc ts-mocha --recursive ./tests/server.spec.ts"
```

Comment:
Good catch.
If this becomes an issue i can move acceptance tests into its own folder later.